content/en/docs/admin/admission-controllers.md

```diff
@@ -140,9 +134,11 @@ enabling this admission controller.
 
 ### EventRateLimit (alpha)
 
-This admission controller mitigates the problem where the API server gets flooded by
+This admission controller is introduced in v1.9 to mitigate the problem where the API server gets flooded by
```

Comment:
Is it necessary to specify the version? That version will be n-2 when 1.11 ships.